src/net_processing.cpp

```diff
@@ -2110,6 +2124,21 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
             nCMPCTBLOCKVersion = 1;
             connman->PushMessage(pfrom, msgMaker.Make(NetMsgType::SENDCMPCT, fAnnounceUsingCMPCTBLOCK, nCMPCTBLOCKVersion));
         }
+
+        if (pfrom->nVersion >= TX_RECONCILIATION_VERSION) {
```

Comment:
Why is this using a protocol version bump again?

It should be possible, but it introduces more confusion than it solves. See for example BIP 61, which was introduced with a version bump, but then lead to confusion such as https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-October/017423.html after removing it from some p2p nodes with a version greater than the BIP 61 version.